packages/driver/cypress/e2e/commands/actions/click.cy.js

```diff
@@ -3339,21 +3303,6 @@ describe('src/cy/commands/actions/click', () => {
         })
       })
 
-      // TODO: remove this after 4.0 when {multiple:true} is no longer default
```

Comment:
TODO from Cypress 4.0, referencing a long-closed issue. TODO done.

Basically, `{multiple: true}` used to be the default, so it was hidden; in Cy >=4.0.0, this is no longer the case, so there's no default value to hide.